save : crashes : topology with containers (cEOS) and vr-routers (vr-csr) :

sudo containerlab save -t ~/clab/EVPN-DCI/EVPN-DCI.clab.yaml
[sudo] password for aristaps:
INFO[0000] Parsing & checking topology file: EVPN-DCI.clab.yaml
panic: runtime error: index out of range [0] with length 0

goroutine 89 [running]:
github.com/srl-labs/containerlab/nodes/vr_csr.(*vrCsr).SaveConfig(0xc0001fcb40, 0x1796cc0, 0xc00003a300, 0x0, 0x0)
	/home/runner/work/containerlab/containerlab/nodes/vr_csr/vr-csr.go:88 +0x22d
github.com/srl-labs/containerlab/cmd.glob..func14.1(0xc0002441a0, 0x1796cc0, 0xc00003a300, 0x17a8208, 0xc0001fcb40)
	/home/runner/work/containerlab/containerlab/cmd/save.go:55 +0x7a
created by github.com/srl-labs/containerlab/cmd.glob..func14
	/home/runner/work/containerlab/containerlab/cmd/save.go:52 +0x289
aristaps@containerlab02:~$

--- a/nodes/node.go
+++ b/nodes/node.go
@@ -106,4 +106,5 @@ var DefaultCredentials = map[string][]string{
 	"vr-vmx":   {"admin", "admin@123"},
 	"vr-vqfx":  {"admin", "admin@123"},
 	"vr-xrv9k": {"clab", "clab@123"},
+	"vr-csr":   {"admin", "admin"},
 }

